app/assets/javascripts/graphs.coffee

```diff
@@ -0,0 +1,155 @@
+$ ->
+  if gon.raw_run isnt `undefined`
```

Comment:
using backticks isn't generally good since that evaluates the expression as raw javascript rather than sticking to coffeescript standards. the idiomatic way to do this in coffeescript is

```coffeescript
if gon.raw_run?
```

if you're ok with it also checking for null / false, or  

```coffeescript
if typeof gon.raw_run isnt 'undefined'
```

otherwise.